gradle.properties

```diff
@@ -7,7 +7,7 @@ slf4jVersion = 1.7.25
 sizeofVersion = 0.3.0
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.4.0
+terracottaPlatformVersion = 5.4-SNAPSHOT
```

Comment:
need new release and merge of https://github.com/Terracotta-OSS/terracotta-platform/pull/450